packages/ocaml-migrate-parsetree/ocaml-migrate-parsetree.1.1.0/opam

```diff
@@ -0,0 +1,21 @@
+opam-version: "1.2"
+maintainer: "frederic.bour@lakaban.net"
+authors: [
+  "Frédéric Bour <frederic.bour@lakaban.net>"
+  "Jérémie Dimino <jeremie@dimino.org>"
+]
+license: "LGPL-2.1"
+homepage: "https://github.com/ocaml-ppx/ocaml-migrate-parsetree"
+bug-reports: "https://github.com/ocaml-ppx/ocaml-migrate-parsetree/issues"
+dev-repo: "https://github.com/ocaml-ppx/ocaml-migrate-parsetree.git"
+doc: "https://ocaml-ppx.github.io/ocaml-migrate-parsetree/"
+tags: [ "syntax" "org:ocamllabs" ]
+build: [
+  ["jbuilder" "build" "-p" name "-j" jobs]
+]
+depends: [
+  "result"
+  "ocamlfind" {build}
```

Comment:
is `ocamlfind` still needed? I looked up the source code and it's only present in the examples Makefiles

Reply:
It was added by this PR: https://github.com/ocaml-ppx/ocaml-migrate-parsetree/pull/13. I'll  submit a PR to remove it upstream, and then a PR to remove it in the opam-repository

Reply:
https://github.com/ocaml-ppx/ocaml-migrate-parsetree/pull/53

Reply:
I remove the ocamlfind dependency in this PR